node/src/rpc.rs

```diff
@@ -119,6 +112,13 @@ where
     Ok(rpc)
 }
 
+#[cfg(feature = "wip")]
+pub fn add_wip_rpc(
+    rpc: RpcExtension,
+) -> Result<RpcExtension, Box<dyn std::error::Error + Send + Sync>> {
+    Ok(rpc)
+}
+
```

Comment:
Why do you need it if your feature will work only under `ready-to-test`?

Reply:
I decided to keep it empty for future usage. For example if we will add RPC to order-book or to another wip feature. To not remove and add this function every time

Reply:
Ok